LICENSE

```diff
@@ -0,0 +1,20 @@
+This package contains a modified version of ca-bundle.crt:
```

Comment:
Plese remove this file. It is included in certifi source and package, and will be picked up in the about section.